src/ShopListFilterBundle/objects/db/ListfilterItems/TPkgShopListfilterItemCategoryTree.class.php

```diff
@@ -30,6 +30,8 @@ class TPkgShopListfilterItemCategoryTree extends TPkgShopListfilterItemMultisele
      * returns std class as tree from the current active category to all available options.
      *
      * @return stdClass
+     * @psalm-suppress UndefinedVariable, InvalidReturnType
+     * @FIXME ???
```

Comment:
This one confused me a bit: `oListItem` is never defined. This method MUST be failing currently.